app/pandas.ipynb

```diff
@@ -436,7 +508,8 @@
    "execution_count": null,
    "outputs": [],
    "source": [
-    "# filter s2 here"
+    "# filter s2 here\n",
+    "s2_odd = s2[s2 % 2 == 0]"
```

Comment:
The filtering logic for `s2` is incorrect. You are currently filtering for even numbers instead of odd numbers. Change the condition to `s2 % 2 != 0` to filter for odd numbers.